src/main/utils/steamOS.ts

```diff
@@ -42,16 +39,28 @@ function getAppId(): string | null {
     return null;
 }
 
-async function execSteamURL(url: string): Promise<void> {
-    await exec(`steam -ifrunning ${url}`);
+export async function execSteamURL(url: string): Promise<void> {
+    // This doesn't allow arbitrary execution despite the weird syntax.
+    await writeFile(
+        join(process.env.HOME || "/home/deck", ".steam", "steam.pipe"),
```

Comment:
async file writes with no synchronisation mechanisms are always a bit worrying. could multiple simultaneous writes lead to a race condition? in this case i could lead to a sandbox escape

Reply:
it's a pipe so I highly doubt

I've also spammed useless unnecessary shit into this pipe and steam just ignores it unless it contains that starting pattern

Reply:
it also always seems to execute immediately so I'm not sure if it's really async? idk